config/7.11.0/common.yaml

```diff
@@ -829,6 +829,40 @@ servers:
                       value: "[[.Jvm.GcContainerOptions]]"
                     #[[end]]
                     ## Jvm config END
+                    ## AMQ Streams BEGIN
+                    #[[if .KafkaExt.ExtDisabled]]
```

Comment:
is it really needed? or is it just a control flag?
Is it a conf that the kieserevr image will expect?

Reply:
ok